src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -190,13 +191,14 @@ def test_set_kern_float_arg(outputdir):
     otrans.apply(sched)
     generated_code = str(psy.gen)
     expected = '''\
-    SUBROUTINE bc_ssh_code_set_args(kernel_obj, nx, a_scalar, ssh_fld, tmask)
+    SUBROUTINE bc_ssh_code_set_args(kernel_obj, nx, a_scalar, ssh_fld, '''+\
```

Comment:
This routine no longer matches the OpenCL implementation of the bc_ssh_code kernel (https://github.com/stfc/PSycloneBench/blob/master/ocean/nemo/nemolite2d/kernels/opencl/boundary_conditions_kern.c).